test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -2699,7 +2699,7 @@ def forward(self, x):
 
         class ComparisonModel(torch.nn.Module):
             def forward(self, x, y):
-                return x.ge(0.5) & y.le(2)
+                return x.lt(1.5) & y.le(2) & x.le(1)
```

Comment:
Can you add test cases for when all inputs are tensors, or all scalars?
Does this cover the later case?

Reply:
added